From e290ae2306b19e065b6580514eed7a53519d062b Mon Sep 17 00:00:00 2001
From: Theodore Tsirpanis <theodore.tsirpanis@tiledb.com>
Date: Fri, 31 May 2024 23:06:58 +0300
Subject: [PATCH] Skip finding the lz4 package.
 
Downstream packagers like Conda and MacPorts need to find lz4 manually, because the packages for lz4 in these environments do not provide CMake configs.
---
 cmake/Modules/FindLZ4_EP.cmake | 5 -----
 1 file changed, 5 deletions(-)
 
diff --git cmake/Modules/FindLZ4_EP.cmake cmake/Modules/FindLZ4_EP.cmake
index b044992998e..b2263217519 100644
--- cmake/Modules/FindLZ4_EP.cmake
+++ cmake/Modules/FindLZ4_EP.cmake
@@ -34,11 +34,6 @@
 # Include some common helper functions.
 include(TileDBCommon)
 
-if(TILEDB_VCPKG)
-  find_package(lz4 REQUIRED)
-  return()
-endif()
-
 # First check for a static version in the EP prefix.
 find_library(LZ4_LIBRARIES
   NAMES
